ru/mail/chrome/messenger/addressbook/abAddressBookNameDialog.dtd

```diff
@@ -36,4 +36,4 @@
  ***** END LICENSE BLOCK ***** -->
 
 <!-- Labels -->
-<!ENTITY name.label      "Имя адресной книги:">
+<!ENTITY name.label      "Имя адресной книги">
```

Comment:
CORRECTIONS NEEDED for line 39: 
+<!ENTITY name.label      "Имя адресной книги:">

All in all, perhaps, we should not trust even MS Language Portal in this case, which contains this translation. General rule is that living creaters have names (имена), while inanimate objects have special names (названия). Thus, we can leave things such as "Имя сервера" or "Имя учетной записи" as they are, and it will be in line with this wrong tradition. Yet, replacing "Имя" by "Название" is recommended.